packages/directlinespeech/__tests__/utilities/fetchSpeechCredentialsWithCache.js

```diff
@@ -2,7 +2,7 @@ import fetch from 'node-fetch';
 
 const TOKEN_URL_TEMPLATE = 'https://{region}.api.cognitive.microsoft.com/sts/v1.0/issueToken';
 
-async function fromWaterBottle() {
+async function fromWaterBottle({ enableInternalHTTPSupport = false }) {
```

Comment:
If the options parameter is `undefined`, `enableInternalHTTPSupport` would inherently be falsy so there is no reason to set the default value to `false`.  

```suggestion
async function fromWaterBottle({ enableInternalHTTPSupport } = {}) {
```